app/controllers/health_controller.rb

```diff
@@ -0,0 +1,8 @@
+class HealthController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:check]
+
+  def check
+    val = ActiveRecord::Base.connection.execute('select 1+2 as val').first['val']
+    render plain: "1+2=#{val}"
```

Comment:
Coming from writing a lot of REST-APIs this is interesting 😂 

I would have expected it to return some JSON here, something like:
```json
{
    "database": "OK"
}
```

But this is also totally fine for now. Possibly with the planned PRTG for during the camp, we could extend to show other subsystems (like Object-Storage reachable, Migrations up to date etc.)
But absolutely ok for now.

Reply:
@cleverer what i like about this check is that you see that the database really works (since it does this calculation on the database). i like the other checks as well 👍

Reply:
I wouldn't bloat this PR and extend that in a separate one, small steps ;)
But if you're absolutely in the flow, just go on ^^